src/components/form/form.types.ts

```diff
@@ -208,18 +215,21 @@ export interface FormComponentOptions {
 /**
  * @public
  */
-export interface FormLayoutOptions<T = FormLayoutType.Default> {
+export interface FormLayoutOptions<
+    T extends FormLayoutType | `${FormLayoutType}` = FormLayoutType.Default,
+> {
     /**
      * The type of layout to use
      */
-    type: T;
+    type?: T;
```

Comment:
This looks like a breaking change, not sure why it's needed :thinking:

Reply:
The reason for this change is because it was "wrong" before.
The layoutOption doesn't actually need a `type` for example if you have a schema with `layout: grid` it's possible for all properties in that schema to set only `colSpan: all` for example. Then you don't have to set the type on that layout.

And are we sure it counts as a breaking change if we make something optional? The other way around would totally be breaking but I think that this direction is fine?

Reply:
Yeah, that's true. Then I will just change the type to Partial in our new Schema instead. Fix coming soon